modules/schematics/BUILD

```diff
@@ -0,0 +1,29 @@
+package(default_visibility = ["//visibility:public"])
+
+load("//tools:defaults.bzl", "ts_library", "npm_package")
+
+ts_library(
+    name = "schematics",
+    srcs = glob(["src/*/*.ts"]),
+    module_name = "@ngrx/schematics",
+)
+
+filegroup(
+    name = "schematics_templates",
+    srcs = glob([
+        "src/*/files/**/*.ts",
+        "src/*/schema.json",
+    ]),
+)
+
+npm_package(
+    name = "npm_package",
```

Comment:
is this published to npm separately from other modules?
just FYI if you need it - there is a `packages` attribute on `ng_package` and `npm_package` that allows you to nest them if needed.

Reply:
Yes, it is published separately.